apps/docs/src/content/docs/drupal/performance/overview.md

```
@@ -23,15 +23,15 @@ These sizes are gzipped. Raw sizes are approximately 2.5–3× larger.
 The full bundle CDN URL is:
 
 ```
-https://cdn.jsdelivr.net/npm/@helixui/library@1.1.2/dist/index.js
+https://cdn.jsdelivr.net/npm/@helixui/library@3.0.0/dist/index.js
 ```
 
 Per-component CDN pattern:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)